schemas/com.snowplowanalytics.snowplow.badrows/enrichment_failures/jsonschema/1-0-0

```diff
@@ -0,0 +1,331 @@
+      "properties": {
+        "raw": {
+          "type": "object",
+          "description": "The raw event extracted from collector payload",
+          "properties": {
+            "vendor": {
+              "type": "string",
+              "description": "Vendor of the adapter that processed this payload, here com.snowplowanalytics.snowplow",
+              "maxLength": 64
+            },
+            "version": {
+              "type": "string",
+              "description": "Version of the adapter that processed this payload",
+              "maxLength": 16
+            },
+            "parameters": {
+              "type": "object",
+              "description": "Tracker protocol parameters"
+            },
+            "contentType": {
+              "type": [ "string", "null" ],
+              "description": "Content type of the payload as detected by the collector",
+              "maxLength": 256
+            },
+            "loaderName": {
+              "type": "string",
+              "maxLength": 32
+            },
+            "encoding": {
+              "type": "string",
+              "description": "Encoding of the collector payload",
+              "maxLength": 32
+            },
+            "hostname": {
+              "type": [ "string", "null" ],
+              "description": "Hostname of the payload as detected by the collector",
+              "maxLength": 8192
+            },
+            "timestamp": {
+              "type": ["string", "null"],
+              "description": "Timestamp at which the payload was collected",
+              "format": "date-time"
+            },
+            "ipAddress": {
+              "type": ["string", "null"],
+              "description": "IP address of the payload as detected by the collector",
+              "maxLength": 128
+            },
+            "useragent": {
+              "type": [ "string", "null" ],
+              "description": "User agent of the payload as detected by the collector",
+              "maxLength": 4096
+            },
+            "refererUri": {
+              "type": [ "string", "null" ],
+              "description": "Referer of the payload as detected by the collector",
+              "maxLength": 8192
+            },
+            "headers": {
+              "type": [ "array", "null" ],
+              "description": "List of the headers part of this collector payload",
+              "items": {
+                "type": "string",
+                "maxLength": 8192
+              }
+            },
+            "userId": {
+              "type": [ "string", "null" ],
+              "description": "Network user id associated with this payload",
+              "format": "uuid"
+            }
+          },
+          "required": [ "vendor", "version", "loaderName", "encoding" ],
+          "additionalProperties": false
+        },
+        "enriched": {
+          "description": "The partially enriched event that resulted in schema violations",
```

Comment:
This will happen when we validate the context added by each enrichment or this is something else?

Reply:
No, this is a payload we had after (failed) enrichments.